diff --git a/data/org.gnome.gitg.appdata.xml.in.in b/data/org.gnome.gitg.appdata.xml.in.in
index 90e9fe63..13bfad3c 100644
--- a/data/org.gnome.gitg.appdata.xml.in.in
+++ b/data/org.gnome.gitg.appdata.xml.in.in
@@ -1,9 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop-application">
   <id>@app-id@</id>
+  <name>gitg</name>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-2.0+</project_license>
   <project_group>GNOME</project_group>
+  <summary>Graphical user interface for git</summary>
   <description>
     <p>
       gitg is a graphical user interface for git. It aims at being a small,

