diff --git i/data/org.gnome.gitg.appdata.xml.in.in w/data/org.gnome.gitg.appdata.xml.in.in
index fc4ab8f3..35efc264 100644
--- i/data/org.gnome.gitg.appdata.xml.in.in
+++ w/data/org.gnome.gitg.appdata.xml.in.in
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Alberto Fanjul <albfan@gnome.org> -->
 <component type="desktop-application">
   <id>@app-id@</id>
   <name>gitg</name>
@@ -13,7 +14,7 @@
       Besides visualization, gitg also provides several utilities to manage your
       repository and commit your work.
     </p>
-    <p>Goals</p>
+    <p>Goals:</p>
     <ul>
       <li>Browse local git repositories</li>
       <li>Clone local and remote git repositories</li>
